packages/cli/test/integration/generators/negative-tests.relation.integration.js

```diff
@@ -0,0 +1,131 @@
+// Copyright IBM Corp. 2019. All Rights Reserved.
+// Node module: @loopback/cli
+// This file is licensed under the MIT License.
+// License text available at https://opensource.org/licenses/MIT
+
+'use strict';
+
+const path = require('path');
+const assert = require('yeoman-assert');
+const testlab = require('@loopback/testlab');
+const fs = require('fs');
+const expect = testlab.expect;
+const TestSandbox = testlab.TestSandbox;
+const generator = path.join(__dirname, '../../../generators/relation');
+const SANDBOX_FILES = require('../../fixtures/relation').SANDBOX_FILES;
+const testUtils = require('../../test-utils');
+
+// Test Sandbox
+const SANDBOX_PATH = path.resolve(__dirname, '..', '.sandbox');
+const MODEL_APP_PATH = 'src/models';
+const CONTROLLER_PATH = 'src/controllers';
+const REPOSITORY_APP_PATH = 'src/repositories';
+const sandbox = new TestSandbox(SANDBOX_PATH);
+
+describe('lb4 relation negative tests', function() {
+  // tslint:disable-next-line:no-invalid-this
+
+  this.timeout(30000);
+
+  beforeEach('reset sandbox', async () => {
+    await sandbox.reset();
+  }); // special cases regardless of the repository type
+
+  context('Execute relation with wrong relation type', () => {
+    it('arguments: set relationType is foo', () => {
+      return expect(
+        testUtils
+          .executeGenerator(generator)
+          .inDir(SANDBOX_PATH, () =>
+            testUtils.givenLBProject(SANDBOX_PATH, {
+              additionalFiles: SANDBOX_FILES,
+            }),
+          )
+          .withArguments('--relationType foo'),
+      ).to.be.rejectedWith(/Incorrect relation type/);
+    });
+    it('prompt: set relationType is foo', () => {
```

Comment:
Similarly here.

Test names should read like an English sentence, where `it` stands for the subject under test as provided by `describe` name.

```ts
it('rejects invalid relation type provided via prompt', () => {
```

Reply:
done